spharpy/spherical.py

```diff
@@ -419,22 +456,32 @@ def spherical_harmonic_basis_real(n_max, coords):
 
 
     """
-    n_coeff = (n_max+1)**2
+    n_coeff = (n_max + 1) ** 2
 
-    basis = np.zeros((coords.csize, n_coeff), dtype=float)
+    basis = np.zeros((coordinates.csize, n_coeff), dtype=float)
 
     for acn in range(n_coeff):
-        order, degree = acn_to_nm(acn)
+        if channel_convention == "fuma":
+            order, degree = fuma_to_nm(acn)
+        else:
+            order, degree = acn_to_nm(acn)
         basis[:, acn] = _special.spherical_harmonic_real(
-            order,
-            degree,
-            coords.colatitude,
-            coords.azimuth)
+            order, degree, coordinates.colatitude, coordinates.azimuth
+        )
+        if normalization == "sn3d":
+            basis[:, acn] *= n3d_to_sn3d_norm(order)
+        elif normalization == "maxN":
+            basis[:, acn] *= n3d_to_maxn(acn)
+        if phase_convention is None:
+            # Condon-Shortley phase term is already included in
+            # the special.spherical_harmonic function
+            # so need to divide by (-1)^m
+            basis[:, acn] /= (-1) ** float(degree)
```

Comment:
I think the spherical_harmonic_real function does not include the CS phase. Please check the issue I created on this and compare.
Technically, there's no difference between multiplying with (-1)^m or dividing by, but the comment is not correct then ;)